frontend/packages/topology/src/extensions/topology.ts

```diff
@@ -51,7 +51,7 @@ namespace ExtensionProperties {
     id: string;
     priority: number;
     quadrant: TopologyDecoratorQuadrant;
-    decorator: CodeRef<TopologyDecoratorGetter>;
+    decorator: () => Promise<TopologyDecoratorGetter>;
```

Comment:
nit: I'm not sure why you are making this change?
Seems to communicate better the intention by using `CodeRef`

Reply:
Above `CodeRef` type was imported from `packages/console-plugin-sdk/src/typings/base.ts` which got modified as well:

```diff
-/**
- * Code reference, resolved to a function that returns the object `T`.
- *
- * TODO: Remove this once https://github.com/openshift/console/pull/7163 gets merged that adds support for dynamic extensions in static plugins.
- */
-export type CodeRef<T> = () => Promise<T>;
```

The `CodeRef` type is declared in `packages/console-dynamic-plugin-sdk/src/types.ts` since it's related to dynamic plugins.

I'll update code to reuse the right `CodeRef` type.

Reply:
@christianvogt Suggestion addressed in the second commit, please have a look.